CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+We use version numbers with three digits such as 1.0.0.
+Any version bump in the last digit is backwards-compatible, in that a model trained with the previous version can still
+be used for translation with the new version.
+Any bump in the second digit indicates potential backwards incompatibilities, e.g. due to changing the architecture or
+simply modifying weight names.
+Note that Sockeye has checks in place to not translate and old model with an incompatible version.
```

Comment:
maybe: checks in place to not translate with a model that was trained with an incompatible version.